src/applications/caregivers/actions/fetchFacilities.js

```diff
@@ -3,23 +3,64 @@ import environment from 'platform/utilities/environment';
 import { apiRequest } from 'platform/utilities/api';
 import content from '../locales/en/content.json';
 
-export const fetchFacilities = async ({
+const joinAddressParts = (...parts) => {
```

Comment:
Pulled these two functions out of the `fetchFacilities` function to clean it up. They are not exported so they aren't exposed outside of the file.